app/views/posts/index.html.erb

```diff
@@ -1,30 +1,42 @@
-<div class="container">
-    <div class="row">
-
-        <div class="col-md-12 d-flex ">
-
-            <img src="" height="50">
-
-            <div class="">
-
-                <h3>My name</h3>
-
-                <span>Number of post: 0</span>
-
+<main class="card">
+      <div class="card-content">
+            <div class="user-image border"></div>
```

Comment:
- [ ] In the posts index page as per the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png), you should also display the `user.photo`